packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -194,6 +194,10 @@ export class SummaryGenerator {
 		cancellationToken: ISummaryCancellationToken,
 		resultsBuilder = new SummarizeResultBuilder(),
 	): ISummarizeResults {
+        this.logger.sendTelemetryEvent({
```

Comment:
You should also put a log when the refresh latest summary is called so we can correlate their order. Currently, the "PendingSummaryNotFound" will be logged but it can happen later.

Reply:
I could but I was hoping to avoid adding logging unless we need to - I bet we would easily see the sumamry log being hit before the pendingSummaryNotFound.  But if you feel strongly about adding logging, I can add it.

Reply:
Since refresh and summarize and both async and they are competing, I was just thinking we should have reliable telemetry that tells us the order in which they happen. If you think current telemetry is enough, I am fine with it :-).